URBNCreativeTestRig/TextDetailTogglesViewController.swift

```diff
@@ -0,0 +1,359 @@
+//
+//  TextDetailTogglesViewController.swift
+//  URBNCreativeTestRig
+//
+//  Created by Jason Grandelli on 11/9/15.
+//  Copyright © 2015 Jason Grandelli. All rights reserved.
+//
+
+import UIKit
+
+class OptionTextField: UITextField {
+    var setTextBlock: () -> (Void) = {}
```

Comment:
Gotcha, that makes sense. In that case this block should pass `self.text` back i guess